src/api/launcher.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+require('babel-register');
```

Comment:
Avoid requiring babel here. Use `--harmony-async-await --require babel-register` in the pm2 config